packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -519,6 +519,11 @@ class BuildSystem {
                      path.contains('.dart_tool');
       });
     }
+    trackSharedBuildDirectory(
+      environment, _fileSystem, buildInstance.outputFiles);
```

Comment:
style nit: if the open paren is on a line without parameters, then the closing paren has to follow suit:
```dart
trackSharedBuildDirectory(
  environment, _fileSystem, buildInstance.outputFiles,
);
```
or
```dart
trackSharedBuildDirectory(environment, _fileSystem, buildInstance.outputFiles);
```

Reply:
Done